media-gfx/kuickshow/kuickshow-0.11.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_HANDBOOK="forceoptional"
+KFMIN=6.0.0
+QTMIN=6.0.0
```

Comment:
Following up my KFMIN bump request: Upstream KF 6.9.0's minimum version is 6.6.0, so a lower one would not make sense here.

Reply:
done